src/strife/d_player.h

```diff
@@ -198,6 +198,13 @@ typedef struct player_s
     // True if secret level has been done.
     //boolean		didsecret;   [STRIFE] Removed this.
 
+    // [AM] Previous position of viewz before think.
+    //      Used to interpolate between camera positions.
+    fixed_t		oldviewz;
```

Comment:
This is an `angle_t` in the other game but I think it's supposed to be a `fixed_t`.

Reply:
I guess it should be the same type as `viewz` independent of the game.

Reply:
It looks like `viewz` is always `fixed_t` so the other games should be updated.

Reply:
Alright, could you please do this in a separate PR?

Reply:
Sure.